versions.yaml

```diff
@@ -199,8 +199,8 @@ externals:
 
   attestation-agent:
     description: "Provide attested key unwrapping for image decryption"
-    url: "https://github.com/confidential-containers/guest-components/"
-    version: "v0.7.0"
+    url: "https://github.com/Xynnn007/guest-components.git"
+    version: "feat-cdh-binary"
```

Comment:
will switch to upstream repo when https://github.com/confidential-containers/guest-components/pull/322 is merged

Reply:
https://github.com/confidential-containers/guest-components/pull/322 is merged, please update to use the upstream tree